jekyll/_posts/2022-12-21-version-20-rc.md

```diff
@@ -261,7 +259,7 @@ For example:
 ```
 
 The types `E1` and `E2` share the names `value1` and `value2`. These are overloaded and the usual overload disambiguation
-is used so that the `E1` or `E2` prefixes can be left out in many cases. This features is most beneficial for independently developed libraries.
+is used so that the `E1` or `E2` prefixes can be left out in many cases. Those features are most beneficial for independently developed libraries.
```

Comment:
```suggestion
is used so that the `E1` or `E2` prefixes can be left out in many cases. These features are most beneficial for independently developed libraries.
```